### PR TITLE
Improve usability if MetaMask is disabled

### LINKED ISSUE #102: Improve usability of ethereum actions if MetaMask is disabled
If MetaMask is disabled, we only prompt the user to enable it. We should make it easier for the user to use an alternate wallet by (for example) providing copy-to-clipboard buttons and a confirmation functionality like the one we use for bitcoin actions (see #101).

Questions:
- How should we handle the case where MetaMask is enabled but the user wants to use a different wallet without bloating the action dialog with buttons and options?

Furthermore, when MetaMask is enabled and the user attempts to send a transaction without logging into their account, the action fails without providing feedback as to what the user should do about the error. The intention was to fix this in #104, but Web3/MetaMask don't provide good enough errors to handle specific cases like this one.

### PR DESCRIPTION
**Pull request by luckysori**
Ethereum actions can now be performed more easily when MetaMask is disabled, by providing copy-to-clipboard buttons and allowing manual confirmation:

Jun26::182704

Jun26::182742

The manual confirmation feature behaves the same way as with Bitcoin actions:

Jun26::192256

The swap page allows the user to repeat an action if they so desire, but the swap page doesn't display this option.

When MetaMask is enabled, the dialog looks exactly the same as before:

Jun26::192956

Resolves #102.

### CODE REVIEW

**Comment by D4nte** [comment]
1. Does the metamask button still appears when it is deactivated?
2. Does the copy button still appear when metamask is activated?

**Comment by luckysori** [comment]
I think my screenshots/gifs weren't clear enough :disappointed: 

>     1. Does the metamask button still appears when it is deactivated?

Yes, so that the user knows MetaMask is an option. There might be a more elegant solution than this, but for now the button is greyed out and a "Please enable MetaMask" tooltip is displayed on hover.

>     2. Does the copy button still appear when metamask is activated?

No. When MetaMask is activated the dialog looks like it did before.

**Comment by bonomat** [comment]
Removed myself and added @thomaseizinger as he is the better judge for this.

File: src/pages/SwapList/LedgerActionDialogBody.tsx [hunk 1]
```diff
@@ -140,34 +176,59 @@ function LedgerActionDialogBody({
...
+              onSuccess={onSuccessMetaMask}
             />
           </DialogActions>
         </React.Fragment>
       );
     }
     case "ethereum-call-contract": {
+      const address = action.payload.contract_address;
+      const data = action.payload.data;
+      const gasLimit = parseInt(action.payload.gas_limit, 16);
```
Comment (da-kami) [inline src/pages/SwapList/LedgerActionDialogBody.tsx]: ❓ `gasLimit` and `data` (above `contract`) are the same in the `ethereum-deploy-contract` case - would it make sense to pull them out and reuse them?

**Pull request merged**